docs/source/guide.rst

```diff
@@ -3,27 +3,113 @@ User Guide
 
 This guide is a starting point for configuring and running the `test-operator`.
 
-Installation
-------------
+
+Running Operator Locally
+------------------------
+This is **quick and easy way** how to experiment with the operator during development of a
```

Comment:
This can be used even if a user doesn't want to experiment with the operator and they just wanna use it.